source/language/openpulse.rst

```diff
@@ -0,0 +1,316 @@
+OpenPulse Grammar
+=================
+
+In addition to OpenQASM instructions, ``defcal`` blocks may contain OpenPulse 
+instructions. Or equivalently stated: OpenPulse instructions cannot appear
+outside of a ``defcal`` block.
```

Comment:
Was the grammar definition removed? I think this is important to start off with.

Reply:
I think it's still in (or is under discussion and would be added back to) `source/language/pulses.rst`